indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -0,0 +1,91 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_track_list(event) %}
+    {% if not event.tracks %}
+        {% call message_box('info', classes='no-rules-placeholder') %}
+            {% trans %}No tracks have been added yet.{% endtrans%}
+        {% endcall %}
+    {% else %}
+        <ul id="track-list" data-url="{{ url_for('.sort_tracks', event) }}">
+            {% for track in event.tracks|sort(attribute='position') %}
+                <li class="track-row i-box" data-id="{{ track.id }}">
+                    <div class="ui-i-box-sortable-handle"></div>
+                    <div class="i-box-header">
+                        <span class="i-box-title">{{ track.title }}</span>
+                        <div class="i-box-buttons toolbar right">
+                            <div class="group">
+                                <a href="#" class="i-button icon-edit" title="{% trans %}Edit track{% endtrans %}"
```

Comment:
@ThiefMaster if you read my suggestion you would see that it didn't say to split it in case of `href="#"` :stuck_out_tongue: